GH-94398: TaskGroup: Fail create_task() during shutdown (#94400)
python/cpython: Misc/NEWS.d/next/Library/2022-06-29-04-42-56.gh-issue-94398.YOq_bJ.rst

> Once an ``asyncio.TaskGroup`` has started shutting down (i.e., at least one task has failed and the task group has started cancelling the remaining tasks), it should not be possible to add new tasks to the task group.

Reviewer: Let's wait for GH-94456 so we can properly refer to `asyncio.TaskGroup` in the Blurb.

```suggestion
Once a :class:`asyncio.TaskGroup` has started shutting down (i.e., at least one task has failed and the task group has started cancelling the remaining tasks), it should not be possible to add new tasks to the task group.
```
